### ISSUE TITLE
face alignment using 106 landmarks

### ISSUE THREAD

**Issue by ZubairKhan001** [deepinsight/insightface#2629]
arcface_dst = np.array(
    [[38.2946, 51.6963], [73.5318, 51.5014], [56.0252, 71.7366],
     [41.5493, 92.3655], [70.7299, 92.2041]],
    dtype=np.float32)

is provided in face_align.py for 5 keypoinst. 
Can we make it work for 106 landmarks, do we have arcface_dst for landmarks?

**Comment by nttstar** [comment]
Yes you only need to specify the locations of the 5 landmarks out of the 106 keypoints.

**Comment by ZubairKhan001** [comment]
OK, thanks, can we remove the extra space that is added to the facial image after alignment.
